src/modules/Dashboard/features/RespondentData/RespondentDataSummary/ReportMenu/ReportMenu.tsx

```diff
@@ -23,8 +23,11 @@ export const ReportMenu = ({
 }: ReportMenuProps) => {
   const { t } = useTranslation();
   const { respondentId } = useParams();
-  const { secretId, nickname } = users.useRespondent(respondentId || '') || {};
-  const respondentLabel = getRespondentLabel(secretId, nickname);
+  const { details } = users.useRespondent(respondentId || '') || {};
+  const respondentLabel = getRespondentLabel(
```

Comment:
DRY, pls use hook to retrieve `respondentLabel` here and in RespondentDataReview...ReviewMenu

Reply:
done